_includes/page__hero.html

```diff
@@ -27,13 +27,6 @@
 >
   {% if page.header.overlay_color or page.header.overlay_image %}
     <div class="wrapper">
-      <h1 class="page__title" itemprop="headline">
-        {% if paginator %}
-          {{ site.title }}{% unless paginator.page == 1 %} {{ site.data.ui-text[site.locale].page | default: "Page" }} {{ paginator.page }}{% endunless %}
-        {% else %}
-          {{ page.title | default: site.title | markdownify | remove: "<p>" | remove: "</p>" }}
-        {% endif %}
-      </h1>
```

Comment:
@pmelchor, do you know from the top of your head if it's okay to change this template file? I want to suppress the page title on the homepage.

Reply:
As a best practice you want to keep a visible and unique <h1> main heading on each page. This is relevant for SEO and other markup related purposes.

If you don't like the design we can tweak that and/or find a different placing for the <h1> on the homepage.

In terms of the template, I think that right now we are only using that one for the homepage.

Reply:
Okay, I'll put it back :).